modin/engines/base/frame/data.py

```diff
@@ -74,6 +78,8 @@ def __init__(
         self._column_widths_cache = column_widths
         self._dtypes = dtypes
         self._filter_empties()
+        if validate_axes is not None and validate_axes is not False:
```

Comment:
Can `validate_axes` be `None`? We can just give a type hint in the method signature for `validate_axes: bool = False`.

Reply:
Yeah, you're right, `validate_axes` is not supposed to be `None`, added type hint here